include/class.organization.php

```diff
@@ -471,6 +471,9 @@ function delete() {
         if (!parent::delete())
             return false;
 
+        // Clear organization from session to avoid refetch failure
+        unset($_SESSION[':Q:orgs'], $_SESSION[':O:tickets']);
```

Comment:
Why are you clearing tickets here?

Reply:
The org is stored there as well. Maybe I shoul just clear the org from tickets?

Reply:
Let me try replicating the issue and revert back to you with thoughts...

Reply:
I can also send you a dump of my session if you want.